operations/tempo-mixin/runbook.md

```diff
@@ -97,6 +97,10 @@ How to **investigate**:
   ```
   sum(rate(tempo_ingester_failed_flushes_total{cluster="...", container="ingester"}[5m])) by (pod)
   ```
+- Check if retries have been failing:
+  ```
+  increase(tempo_ingester_flush_failed_retries_total) > 0
```

Comment:
I didn't even know this metric existed. I'm actually a big concerned this will just complicate/confuse the situation. Is there any difference between a bunch of failed flushes and a bunch of failed retries from an operational point of view?